treeherder/etl/runnable_jobs.py

```diff
@@ -127,11 +128,14 @@ def _taskcluster_runnable_jobs(project, decision_task_id):
     if not decision_task_id:
         return ret
 
-    tc_graph_url = settings.TASKCLUSTER_TASKGRAPH_URL.format(task_id=decision_task_id)
+    tc_graph_url = settings.TASKCLUSTER_PUBLIC_ARTIFACTS_URL.format(
+        task_id=decision_task_id,
+        filename='runnable-jobs.json.gz'
+    )
     validate = URLValidator()
     try:
         validate(tc_graph_url)
-        tc_graph = fetch_json(tc_graph_url)
+        tc_graph = fetch_gzip_json(tc_graph_url)
```

Comment:
The requests package automatically sets an appropriate gzip `Accept` header and handles decompression, so it should be possible to just use `fetch_json()` directly.